src/v7/transforms/route-ids.ts

```diff
@@ -0,0 +1,37 @@
+import { getLineEndings } from '../../util';
+
+export default function(file: any, api: any) {
+	let quote: string | undefined;
+
+	if (file.path && /^routes\.ts(x)?$/.test(file.path)) {
```

Comment:
Will this do `src/routes`, is the `src` part implied?

Reply:
Nope, that's a mistake, which also made me realize I hadn't tested this one against an actual app. Removed the caret and tested all three combined against a fresh `dojo create app` 👍